modules/packages/LinearAlgebra.chpl

```diff
@@ -1978,8 +1979,8 @@ module Sparse {
 
   /* Compute the dot-product */
   proc _array.dot(A: []) where isCSArr(A) || isCSArr(this) {
-    import LinearAlgebra;
-    return LinearAlgebra.Sparse.dot(this, A);
+    import LinearAlgebra.Sparse;
+    return Sparse.dot(this, A);
```

Comment:
This concerns me a little, the Github display seems to indicate that we are currently within the Sparse module.  I think we should still be able to use qualified naming with the current module name